BrowserKit/Sources/TabDataStore/WindowData.swift

```diff
@@ -6,7 +6,6 @@ import Foundation
 
 public struct WindowData: Codable {
     public let id: UUID
-    public let isPrimary: Bool
```

Comment:
@OrlaM Let me know if you want to discuss this change (or anything else). Re: `isPrimary`, I can't see any reason so far to keep this on `WindowData`, though I'm not 100% certain if it should remain for the purposes of tab synchronization with app services (either now or in the future). I will dig into that further before this is merged, but if you have any background context/knowledge on that already please LMK. Right now the concept of `isPrimary` almost doesn't make sense on iOS because multiple windows can effectively be active at the same time (at least from the user's perspective).

Reply:
This is not needed so feel free to kill it.
App services won't need it either, there is no concept of a window in sync and isn't likely to be anytime soon.